docs/resources/aws_s3_bucket_object.md

```diff
@@ -0,0 +1,81 @@
+---
+title: About the aws_s3_bucket_object Resource
+---
+
+# aws_s3_bucket_object
```

Comment:
Now that we're in core InSpec, we need to escape all underscores that aren't quoted.  So here we need `aws\_s3\_bucket\_object`